package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "event-sidecar",
+  "version": "1.0.0",
+  "description": "Event Handling sidecar for FRMS Processors",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "tsx src/index.ts",
+    "build": "tsc --project tsconfig.json",
+    "start": "node -r dotenv/config build/index.js"
+  },
+  "author": "Jean-Pierre Nell",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@frmscoe/frms-coe-lib": "https://github.com/rtkay123/coe-lib.git",
```

Comment:
May let point to the internal library on GitHub before approval

Reply:
Right good catch.